app/src/main/java/com/nadosunbae_android/app/presentation/ui/mypage/MyPageSettingFragment.kt

```diff
@@ -64,10 +66,30 @@ class MyPageSettingFragment : BaseFragment<FragmentMyPageSettingBinding>(R.layou
             mainViewModel.mypageFragmentNum.value = 2
         }
 
-        //탈퇴 activity
+        //탈퇴 dialog
         binding.textMypageSettingQuit.setOnClickListener {
-            val intentQuit = Intent(getActivity(), QuitActivity::class.java)
-            startActivity(intentQuit)
+            val dialog = getActivity()?.let { it1 -> QuitAlertCustomDialog(it1) }
+            dialog?.showDialog()
+            dialog?.initBtnClickDialog(R.layout.activity_quit_alert_custom_dialog)
+
```

Comment:
```suggestion
          val dialog = QuitAlertCustomDialog(requireActivity())
          dialog.showDialog()
```
이렇게 적어도 뭔가 작동 될 것 같은데